### PR TITLE
MPGDDIRC: fix compilation error and some format strings

### PR DESCRIPTION
**Pull request by veprbl**
```output
src/MPGDDIRC_geo.cpp:126:52: error: cannot pass object of non-trivial type 'std::__1::string' (aka 'basic_string<char, char_traits<char>, allocator<char>>') through variadic function; call will abort at runtime [-Wnon-pod-varargs]
        printout(DEBUG, "MPGDDIRC_geo", "gas: %f", comp_name);
                                                   ^
```
### What kind of change does this PR introduce?
- [x] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### CODE REVIEW

**Review by mposik1983 — DISMISSED** [review]

File: src/MPGDDIRC_geo.cpp [hunk 1]
```diff
@@ -123,13 +123,13 @@ static Ref_t create_MPGDDIRC_geo(Detector& description, xml_h e, SensitiveDetect
         max_component_length = box_length;
         gas_thickness += x_comp.thickness();
         c_box = {box_width / 2, box_length / 2, x_comp.thickness() / 2};
-        printout(DEBUG, "MPGDDIRC_geo", "gas: %f", comp_name);
+        printout(DEBUG, "MPGDDIRC_geo", "gas: %s", comp_name.c_str());
```
Comment (wdconinc) [inline src/MPGDDIRC_geo.cpp:126]: Hmm. it seems like our CI should have caught this... Maybe gcc vs clang? Suggestions on how to avoid?

**Review by wdconinc — DISMISSED** [review]

File: src/MPGDDIRC_geo.cpp (hunk 1 above)
Comment (veprbl) [reply src/MPGDDIRC_geo.cpp:126]: This was using clang

Comment (wdconinc) [reply src/MPGDDIRC_geo.cpp:126]: #215 should hopefully avoid this

**Review by wdconinc — APPROVED** [review]

**Comment by wdconinc** [comment]
Fixed two more unused variables which prevented `-Werror` compilation with clang (and clang-format for pre-commit). This will now allow #215 to succeed.

**Pull request merged**